back-end/routes/new-accounts.js

```diff
@@ -0,0 +1,49 @@
+var express = require('express');
+const fetch = require('node-fetch');
+const settings = require('../settings');
+const mongoose = require('mongoose');
+var router = express.Router();
+
+var accountsSchema = new mongoose.Schema(
```

Comment:
should we move this to a separate file for collection schemas?

Reply:
I already have setup a new file for schema in another folder. This is only for testing purpose.